cogs/slowmode.py

```diff
@@ -8,9 +8,29 @@
 
 import util
 
+import time
+
 HELPER_ROLE = os.getenv('HELPER_ROLE')
 MOD_ROLE = os.getenv('MOD_ROLE')
 
+CHANNELS = [190327462087884811, 766335071590023247]
+
+message_cache = {}
+previous_delays = {}
+last_updated = 0
```

Comment:
any reason these variables should be global and not as attributes in the class?

Reply:
added aas attributes to the class